tools/build.py

```diff
@@ -194,6 +195,30 @@ def mkimd(self, name, src: Target = None):
     )
 
 
+@Rule
+def img2os5(self, name, src: Target = None):
+    simplerule(
+        replaces=self,
+        ins=[src],
+        outs=[f"={name}.os5"],
```

Comment:
Using `{self.shortname}` here produces nicer results (even though the filename here doesn't really matter).

Reply:
I'm not sure what you mean. I can't find any reference to .shortname in the other build scripts.